### PR TITLE
Nit Picky Clean Up

### LINKED ISSUE #342: Don't show prompt in code blocks where participant needs to copy and paste
*here's a convention in Markdown that states you should only prefix command with a prompt or dollar sign if you're showing output. Otherwise, you're just making copying and pasting more difficult for a user of that documentation. I ran into this running the IAM CLI commands in the first section. See http://www.cirosantilli.com/markdown-style-guide/#dollar-signs-in-shell-code for the rationale. Markdownlint (https://github.com/markdownlint/markdownlint) could identify places where this is happening and make it easy to clean up if you were interesting in adopting this style.

### PR DESCRIPTION
**Pull request by christopherhein**
> WIP, please do not merge.

**Why:**

* reduce some very basic things that slow people down

**This change addresses the need by:**

* closes #342

Signed-off-by: Christopher Hein <me@christopherhein.com>

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: 01-path-basics/102-your-first-cluster/readme.adoc [hunk 1]
```diff
@@ -92,32 +92,30 @@ The `--zones` option is also used to distribute the worker nodes. The number of
 
 It will take 5-8 minutes for the cluster to be created. Validate the cluster:
 
-```
-$ kops validate cluster
-Using cluster from kubectl context: example.cluster.k8s.local
+    $ kops validate cluster
```
Comment (arun-gupta) [inline 01-path-basics/102-your-first-cluster/readme.adoc]: `$` still exists?

File: 01-path-basics/102-your-first-cluster/readme.adoc [hunk 2]
```diff
@@ -197,25 +195,23 @@ Once the `kops create cluster` command is issued, it provisions the EC2 instance
 
 Wait for 5-8 minutes and then the cluster can be validated as shown:
 
-```
-$ kops validate cluster
-Using cluster from kubectl context: example.cluster.k8s.local
+    $ kops validate cluster
```
Comment (arun-gupta) [inline 01-path-basics/102-your-first-cluster/readme.adoc]: `$` still exists?

File: 01-path-basics/102-your-first-cluster/readme.adoc [hunk 3]
```diff
@@ -248,57 +244,57 @@ Save the changes and exit the editor. Kubernetes cluster needs to re-read the co
 
 NOTE: This process can easily take 30-45 minutes. Its recommended to leave the cluster without any updates during that time.
 
-  $ kops rolling-update cluster --yes
-  Using cluster from kubectl context: example.cluster.k8s.local
+    $ kops rolling-update cluster --yes
```
Comment (arun-gupta) [inline 01-path-basics/102-your-first-cluster/readme.adoc]: `$` still exists?

File: 01-path-basics/102-your-first-cluster/readme.adoc [hunk 4]
```diff
@@ -248,57 +244,57 @@ Save the changes and exit the editor. Kubernetes cluster needs to re-read the co
...
 
 This command will first stop one master node in the cluster, re-read the configuration information and start that master. Then it will do the same for rest of the master nodes. And then it will repeat that for each worker node in the cluster. After all the server and worker nodes have been restarted, the rolling update of the cluster is complete.
 
 Let's verify that the attributes are now successfully passed to the API server. Get the list of pods for the API server using the command shown:
 
-  $ kubectl get pods --all-namespaces | grep kube-apiserver
-  kube-system   kube-apiserver-ip-172-20-117-32.ec2.internal            1/1       Running   0          7m
-  kube-system   kube-apiserver-ip-172-20-62-108.ec2.internal            1/1       Running   6          16m
-  kube-system   kube-apiserver-ip-172-20-79-64.ec2.internal             1/1       Running   2          12m
+    $ kubectl get pods --all-namespaces | grep kube-apiserver
```
Comment (arun-gupta) [inline 01-path-basics/102-your-first-cluster/readme.adoc]: `$` still exists?

File: 01-path-basics/102-your-first-cluster/readme.adoc [hunk 5]
```diff
@@ -364,48 +360,46 @@ Alternatively, you may leave off the `--yes` option from the `kops create cluste
...
+    kops update cluster example.cluster.com --yes
 
 Once the `kops create cluster` or `kops update cluster` command is issued with the `--yes` flag,, it provisions the EC2 instances, setup Auto Scaling Groups, IAM users, security groups, and install Kubernetes on each node, configures master and worker nodes. This process can take a few minutes based upon the number of master and worker nodes.
 
 Wait for 5-8 minutes and then the cluster can be validated as shown:
 
-```
-$ kops validate cluster --name=example.cluster.com
-Validating cluster example.cluster.com
+    $ kops validate cluster --name=example.cluster.com
```
Comment (arun-gupta) [inline 01-path-basics/102-your-first-cluster/readme.adoc]: `$` still exists?

File: 01-path-basics/102-your-first-cluster/readme.adoc [hunk 6]
```diff
@@ -364,48 +360,46 @@ Alternatively, you may leave off the `--yes` option from the `kops create cluste
...
-Your cluster example.cluster.com is ready
-```
+    Your cluster example.cluster.com is ready
 
 Verify the client and server version:
 
-  $ kubectl version
-  Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.1", GitCommit:"f38e43b221d08850172a9a4ea785a86a3ffa3b3a", GitTreeState:"clean", BuildDate:"2017-10-12T00:45:05Z", GoVersion:"go1.9.1", Compiler:"gc", Platform:"darwin/amd64"}
-  Server Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.4", GitCommit:"793658f2d7ca7f064d2bdf606519f9fe1229c381", GitTreeState:"clean", BuildDate:"2017-08-17T08:30:51Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
+    $ kubectl version
```
Comment (arun-gupta) [inline 01-path-basics/102-your-first-cluster/readme.adoc]: `$` still exists?

File: 01-path-basics/102-your-first-cluster/readme.adoc [hunk 7]
```diff
@@ -418,30 +412,28 @@ A multi-master cluster can be created by using the `--master-count` option and s
...
-ip-172-20-127-147.ec2.internal  node  True
-ip-172-20-35-38.ec2.internal  node  True
-ip-172-20-47-199.ec2.internal node  True
-ip-172-20-61-207.ec2.internal master  True
-ip-172-20-75-78.ec2.internal  master  True
-ip-172-20-94-216.ec2.internal node  True
-
-Your cluster example.cluster.com is ready
-```
+  $ kops validate cluster --name=example.cluster.com
```
Comment (arun-gupta) [inline 01-path-basics/102-your-first-cluster/readme.adoc]: `$` still exists?

File: 01-path-basics/103-kubernetes-concepts/readme.adoc [hunk 8]
```diff
@@ -62,118 +62,113 @@ Get the list of running pods:
...
-  ----    ------                 ----  ----                                                 -------
-  Normal  Scheduled              46s   default-scheduler                                    Successfully assigned nginx-4217019353-pmkzb to ip-172-20-87-91.us-east-2.compute.internal
-  Normal  SuccessfulMountVolume  46s   kubelet, ip-172-20-87-91.us-east-2.compute.internal  MountVolume.SetUp succeeded for volume "default-token-cqht0"
-  Normal  Pulling                46s   kubelet, ip-172-20-87-91.us-east-2.compute.internal  pulling image "nginx"
-  Normal  Pulled                 42s   kubelet, ip-172-20-87-91.us-east-2.compute.internal  Successfully pulled image "nginx"
-  Normal  Created                42s   kubelet, ip-172-20-87-91.us-east-2.compute.internal  Created container
-  Normal  Started                42s   kubelet, ip-172-20-87-91.us-east-2.compute.internal  Started container
-
-```
+    $ kubectl describe pod/<pod-name>
```
Comment (arun-gupta) [inline 01-path-basics/103-kubernetes-concepts/readme.adoc]: `$` still exists?

File: 01-path-basics/103-kubernetes-concepts/readme.adoc [hunk 9]
```diff
@@ -62,118 +62,113 @@ Get the list of running pods:
...
-kube-proxy-ip-172-20-63-150.us-east-2.compute.internal                1/1       Running   0          28d
-kube-proxy-ip-172-20-71-14.us-east-2.compute.internal                 1/1       Running   0          28d
-kube-proxy-ip-172-20-87-91.us-east-2.compute.internal                 1/1       Running   0          28d
-kube-proxy-ip-172-20-94-153.us-east-2.compute.internal                1/1       Running   0          28d
-kube-scheduler-ip-172-20-124-26.us-east-2.compute.internal            1/1       Running   0          28d
-kube-scheduler-ip-172-20-52-35.us-east-2.compute.internal             1/1       Running   0          28d
-kube-scheduler-ip-172-20-94-153.us-east-2.compute.internal            1/1       Running   0          28d
-tiller-deploy-1114875906-k2pj2                                        1/1       Running   0          28d
-```
+    $ kubectl get pods --namespace kube-system
```
Comment (arun-gupta) [inline 01-path-basics/103-kubernetes-concepts/readme.adoc]: `$` still exists?

File: 01-path-basics/103-kubernetes-concepts/readme.adoc [hunk 10]
```diff
@@ -186,45 +181,45 @@ A Pod is the smallest deployable unit that can be created, scheduled, and manage
...
-	    name: nginx-pod
-	    labels:
-	      name: nginx-pod
-	  spec:
-	    containers:
-	    - name: nginx
-	      image: nginx:latest
-	      ports:
-	      - containerPort: 80
+    $ cat pod.yaml
```
Comment (arun-gupta) [inline 01-path-basics/103-kubernetes-concepts/readme.adoc]: `$` still exists?

File: 01-path-basics/103-kubernetes-concepts/readme.adoc [hunk 11]
```diff
@@ -186,45 +181,45 @@ A Pod is the smallest deployable unit that can be created, scheduled, and manage
...
 
 image::nginx-pod-default-page.png[]
 
 If the containers in the pod generate logs, then they can be seen using the command shown:
 
-	$ kubectl logs nginx-pod
-	127.0.0.1 - - [03/Nov/2017:17:33:30 +0000] "GET / HTTP/1.1" 200 612 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36" "-"
-	127.0.0.1 - - [03/Nov/2017:17:33:32 +0000] "GET /favicon.ico HTTP/1.1" 404 571 "http://localhost:8080/" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36" "-"
-	2017/11/03 17:33:32 [error] 5#5: *2 open() "/usr/share/nginx/html/favicon.ico" failed (2: No such file or directory), client: 127.0.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost:8080", referrer: "http://localhost:8080/"
+    $ kubectl logs nginx-pod
```
Comment (arun-gupta) [inline 01-path-basics/103-kubernetes-concepts/readme.adoc]: `$` still exists?

File: 01-path-basics/103-kubernetes-concepts/readme.adoc [hunk 12]
```diff
@@ -253,109 +248,109 @@ CPU can be requested in _cpu units_. 1 cpu unit is equivalent 1 AWS vCPU. It can
 
 By default, a container in a pod is allocated no memory request/limit and 100m CPU request and no limit. This can be verified using the previously started pod:
 
-	$ kubectl get pod/nginx-pod -o jsonpath={.spec.containers[].resources}
-	map[requests:map[cpu:100m]]
+    $ kubectl get pod/nginx-pod -o jsonpath={.spec.containers[].resources}
```
Comment (arun-gupta) [inline 01-path-basics/103-kubernetes-concepts/readme.adoc]: `$` still exists?

File: 01-path-basics/103-kubernetes-concepts/readme.adoc [hunk 13]
```diff
@@ -253,109 +248,109 @@ CPU can be requested in _cpu units_. 1 cpu unit is equivalent 1 AWS vCPU. It can
...
-	    resources:
-	      limits:
-	        memory: "200Mi"
-	        cpu: 2
-	      requests:
-	        memory: "100Mi"
-	        cpu: 1
-	    ports:
-	    - containerPort: 80
+    $ cat pod-resources.yaml
```
Comment (arun-gupta) [inline 01-path-basics/103-kubernetes-concepts/readme.adoc]: `$` still exists?

**Comment by christopherhein** [comment]
Instead of replying to reach, yes @jpignata in #342 pointed to a markdown standard where by you use `$` when you show the output and so the user knows what to copy, and when you don’t show an output leave the `$` out. 

We’re using adoc but the standard still feels nice and makes reading it a little easier.

**Comment by christopherhein** [comment]
I’m still working through the rest of the files so please don’t merge :+1:

Would be curious to hear your thoughts @arun-gupta

**Comment by arun-gupta** [comment]
I like some way to differentiate output from the command. Is there a way to bold the text inside code formatting? 

Wondering what would @mojavelinux recommend?

**Comment by christopherhein** [comment]
There isn't a way to bold the code, I would have liked that too. I think the `$` helps, plus there is tooling (https://github.com/markdownlint/markdownlint) to validate files

**Comment by christopherhein** [comment]
@arun-gupta I just finished a comb of all the files if you want to give it a once over, just trying to get them all using the same general style for code blocks, and when showing return values just having them under the command instead of broken into multiple lines.

**Comment by mojavelinux** [comment]
There actually is a way to bold the code. But if you use custom formatting in a verbatim block, you cannot also have syntax highlighting. It needs to be one or the other.

You enable inline formatting by adding the quotes substitution:

```
[subs=+quotes]
 *$* command
```

Keep in mind, the quotes (which is inline formatting) will now be applied to the whole content of the verbatim block.

For more details, see http://asciidoctor.org/docs/user-manual/#incremental-substitutions

(If you want more control, you can provide a replacement substitution list, but do not forget to add "specialchars" or else you could get illegal HTML characters in the output).

**Comment by christopherhein** [comment]
awesome thanks @mojavelinux, I'll take a look and see how that renders

**Comment by mojavelinux** [comment]
Btw, the default Asciidoctor stylesheet ships with support for an output block. An output block has inverted colors (white on black).

Here's how you do it:

```
[.output]
....
Output of a command goes here.
....
```

If you use your own stylesheet, of course you can style the `output` class however you like.

**Comment by christopherhein** [comment]
does that work with Github's render?

**Comment by christopherhein** [comment]
does that work with Github's render? or just Asciidoctor? @mojavelinux

**Comment by mojavelinux** [comment]
GitHub only applies a generic stylesheet to content files, so any feature that relies on CSS doesn't work. The only way to control the appearance is by relying on default HTML styling (which is why bold and italic text work fine).

**Comment by mojavelinux** [comment]
But you can view any content file on GitHub in full fidelity using the gist.asciidoctor.org view. For example: http://gist.asciidoctor.org/?github-aws-samples%2Faws-workshop-for-kubernetes%2F%2Freadme.adoc

This tool is great for sharing links.

**Comment by mojavelinux** [comment]
(I'm aware there is no https yet for gist.asciidoctor.org. That's on my list to do).

**Comment by mojavelinux** [comment]
(You could also run docgist on your own servers as it's just a static HTML page).

**Pull request closed without merging**